cri/v1alpha1/cri_utils.go

```diff
@@ -770,3 +802,115 @@ func parseUserFromImageUser(id string) string {
 	// no group, just return the id
 	return id
 }
+
+func (c *CriManager) getContainerMetrics(meta *mgr.Container, stats *containerdtypes.Metric) (*runtime.ContainerStats, error) {
+	var usedBytes, inodesUsed uint64
+	cs := &runtime.ContainerStats{}
+
+	sn, err := c.SnapshotStore.Get(meta.ID)
```

Comment:
snapshot id may not equals container ID later, comments for instension

Reply:
A comment has been added.